cognite/client/experimental.py

```diff
@@ -13,4 +12,3 @@ class CogniteClient(Client):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.model_hosting = ModelHostingAPI(self._config, api_version="playground", cognite_client=self)
```

Comment:
discussed with @hhenriques and we decided to remove MH here as well, and just have experimental inherit from beta - to be removed on the next version bump. Probably would be good to give a warning in init as well to use beta.